Tests/XCUITests/HistoryTests.swift

```diff
@@ -395,60 +399,59 @@ class HistoryTests: BaseTestCase {
-        }
-        // Go to 'google.com' to create a recent history entry.
-        navigateToGoogle()
+        // Visit a page to create a recent history entry.
+        navigateToPage()
         navigator.performAction(Action.ClearRecentHistory)
         // Recent data will be removed after calling tapOnClearRecentHistoryOption(optionSelected: "Today").
         // Older data will not be removed
         tapOnClearRecentHistoryOption(optionSelected: "Today")
         for entry in oldHistoryEntries {
             XCTAssertTrue(app.tables.cells.staticTexts[entry].exists)
         }
-        XCTAssertFalse(app.tables.cells.staticTexts["Google"].exists)
-        
+        XCTAssertFalse(app.staticTexts["Today"].exists)
+        XCTAssertTrue(app.staticTexts["Older"].exists)
+
         // Begin Test for Today and Yesterday
-        // Go to 'goolge.com' to create a recent history entry.
-        navigateToGoogle()
+        // Visit a page to create a recent history entry.
+        navigateToPage()
         navigator.performAction(Action.ClearRecentHistory)
         // Tapping "Today and Yesterday" will remove recent data (from yesterday and today).
         // Older data will not be removed
         tapOnClearRecentHistoryOption(optionSelected: "Today and Yesterday")
         for entry in oldHistoryEntries {
             XCTAssertTrue(app.tables.cells.staticTexts[entry].exists)
         }
-        XCTAssertFalse(app.tables.cells.staticTexts["Google"].exists)
-        
+        XCTAssertFalse(app.staticTexts["Today"].exists)
+        XCTAssertTrue(app.staticTexts["Older"].exists)
+
         // Begin Test for Everything
-        // Go to 'goolge.com' to create a recent history entry.
-        navigateToGoogle()
+        // Visit a page to create a recent history entry.
+        navigateToPage()
         navigator.performAction(Action.ClearRecentHistory)
         // Tapping everything removes both current data and older data.
         tapOnClearRecentHistoryOption(optionSelected: "Everything")
         for entry in oldHistoryEntries {
-            mozWaitForElementToNotExist(app.tables.cells.staticTexts[entry], timeoutValue: 10)
+            mozWaitForElementToNotExist(app.tables.cells.staticTexts[entry])
 
         XCTAssertFalse(app.tables.cells.staticTexts[entry].exists, "History not removed")
         }
-        XCTAssertFalse(app.tables.cells.staticTexts["Google"].exists)
-    }*/
+        XCTAssertFalse(app.staticTexts["Today"].exists)
+        XCTAssertFalse(app.staticTexts["Older"].exists)
+    }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2306890
     // Smoketest
     func testDeleteHistoryEntryBySwiping() throws {
-        navigateToGoogle()
+        navigateToPage()
         navigator.goto(LibraryPanel_History)
         waitForExistence(app.cells.staticTexts["http://example.com/"], timeout: TIMEOUT)
         if processIsTranslatedStr() == m1Rosetta {
```

Comment:
Since we have now tests running on native M1 simulators, we can remove this if condition.
Also, the swipe gesture works now on simulators, so no need to keep this

Reply:
Updated.